model/utils.py

```diff
@@ -248,7 +243,7 @@ def str_to_args(args_str, order):
     return parsed
 
 
-def parse_packets(flp, packet_size_B, direction="data"):
+def parse_packets(flp, unfair_idx, packet_size_B, direction="data"):
```

Comment:
Let's change `unfair_idx` to `flw_idx`, since this can technically be used to parse any of the flows.